src/Cache/CacheItem.php

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
I have no comments to the functionality, but the class doesn't follow the interface though.